WickedEngine/wiGraphicsDevice_Vulkan.h

```diff
@@ -46,15 +46,13 @@ namespace wiGraphics
 		VkQueue copyQueue = VK_NULL_HANDLE;
 
 		VkPhysicalDeviceProperties2 properties2 = {};
-		VkPhysicalDeviceVulkan11Properties properties_1_1 = {};
```

Comment:
also, pls don't remove these (`properties_1_1` and `features_1_1`) , it's useful for looking into these properties with debugger currently, and if we want something from here in the future.